### ISSUE TITLE
Install failed:  unable to fetch clibs/logger.h

### ISSUE THREAD

**Issue by weakish** [clibs/clib-validate#3]
```sh
# clib install clib-validate
       fetch : clibs/clib-validate:package.json
       fetch : clibs/logger:package.json
       fetch : clibs/logger.h:logger.h
       error : unable to fetch clibs/logger.h:logger.h
```

**Comment by yorkie** [comment]
@weakish It works for me, would you please try install again?

**Comment by weakish** [comment]
@yorkie Maybe you have installed `clibs/logger` before?

Manually running `make` (with dependent c sources fetched via `clib`) successfully installed  `clib-validate`.

**Comment by yorkie** [comment]
> @yorkie Maybe you have installed clibs/logger before?

Nope, when the tty shows the "fetch" line, I feel that the clib is doing request from GitHub. BTW, what's your clib version?

**Comment by weakish** [comment]
> BTW, what's your clib version?

@yorkie 1.5.0

**Comment by yorkie** [comment]
@weakish Awesome catch! I reproduced this issue in my side and fixed at #4, this issue is introduced at v1.5.0, then v1.4.2 is working if you are super urgent to use this package :-)
